11ty/definitions/third-world-country.md

```diff
@@ -0,0 +1,20 @@
+---
+title: third-world country
+slug: third-world-country
+defined: true
+speech: noun
+---
+
+a problematic term used to refer to a country that doesn’t quite align with Western ideology or the Western image of society. (_Also common are terms such as “third-world” (adj.), “first-world”, “developing/developed nations”, etc._)
+
+## History
+
+“Three worlds” terminology originally arose during the cold war to define countries that were aligned with NATO (“1st world”), aligned with the Warsaw Pact (“2nd world”), or non-aligned (“3rd world”). Because many third-world countries were economically poor, it became common to refer to poor countries as third-world.
```

Comment:
```suggestion
“Three worlds” terminology originally arose during the cold war to define countries that were aligned with North Atlantic Treaty Organization (NATO) (“1st world”), aligned with the Warsaw Pact (“2nd world”), or non-aligned (“3rd world”). Because many third-world countries were economically poor, it became common to refer to poor countries as third-world.
```

Reply:
I think some additional insight here as to which countries were listed in each of those three designations would help to reiterate the point.